pkg/services/govmomi/power_test.go

```diff
@@ -0,0 +1,351 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package govmomi
+
+import (
+	goctx "context"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/vim25"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	infrav1 "sigs.k8s.io/cluster-api-provider-vsphere/apis/v1beta1" //nolint
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake" //nolint
```

Comment:
Use GCI to format instead of using //nolint

https://github.com/daixiang0/gci

Reply:
This doesn't work for my case but `make lint-fix` fixes it.
Thanks.

Reply:
make lint-fix is basically golangci-lint triggering gci with "-fix" (among other things)

Reply:
Let me check my local toolchain setting later.